app/controllers/mixins/ems_common_angular.rb

```diff
@@ -475,7 +475,7 @@ def get_hostname_from_routes(ems, endpoint_hash, token)
       client = ems.class.raw_connect(endpoint.hostname, endpoint.port,
                                      :service => :openshift, :bearer => token, :ssl_options => ssl_options)
       client.get_route('hawkular-metrics', 'openshift-infra').try(:spec).try(:host)
-    rescue KubeException, OpenSSL::SSL::SSLError => e
+    rescue StandardError => e
```

Comment:
I think that is the same as `rescue => bang`
any way I think `rescue KubeException, OpenSSL::SSL::OpenSSLError => e` is better for us. 
We would not like this to catch an `undefined method` for example.